vtexdocs/devportal
# Feature/subscribe newsletter section

#### What is the purpose of this pull request?

To add the subscribe newsletter section component.

Documentation [here.](https://www.notion.so/vtexhandbook/Criar-adicionar-pagina-de-VTEX-IO-e-landing-page-o-componente-de-Newsletter-fc2bb6f63b01443f951a3206a66ccb95)

#### What problem is this solving?

The absence of the subscribe newsletter section

#### How should this be manually tested?
Check the [landing page](https://deploy-preview-67--elated-hoover-5c29bf.netlify.app/) and the [vtex io](https://deploy-preview-67--elated-hoover-5c29bf.netlify.app/docs/vtex-io) page against [figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=129%3A28).

#### Screenshots or example usage

#### Types of changes
